pyartcd/pyartcd/pipelines/build_microshift_bootc.py

```
@@ -197,27 +205,117 @@ async def _run_pipeline(self):
+        builds: dict[str, KonfluxBuildRecord] = {}
+        for variant in variants:
+            build = await self._rebase_and_build_bootc(variant)
+            if build:
+                builds[variant["image_name"]] = build
+                self._logger.info("Bootc image build for %s: %s", variant["image_name"], build.nvr)
+
+        if not builds:
+            raise ValueError(f"No bootc image builds produced for assembly {self.assembly}")
+
+        # Sync and publish each build
+        for image_name, bootc_build in builds.items():
```

Ruff (0.15.15)

[warning] Loop control variable `image_name` not used within loop body

(B007)